Makefile

```diff
@@ -0,0 +1,39 @@
+REPO_NAME = ccas
+IMAGE_NAME = $(REPO_NAME)/docusaurus:latest
+CONTAINER_NAME = open-edu-hub-$(REPO_NAME)-bash
+OUTPUT_DIR = $$PWD/.output/ccas/
+
+.PHONY: all buildimg build serve run_bash enter_bash stop_bash clean cleanall
+
+all: build
+
+buildimg:
+	docker build -f ./Dockerfile --tag $(IMAGE_NAME) .
+
+build: buildimg
+	@echo "Building content. This will take a while (several minutes) ..."
+	@echo "After the build, run `make serve`"
```

Comment:
`make serve` is not needed. Running the container already starts serving port 8080. No idea why, though.

Reply:
Are you sure? Does it work for you?

Reply:
Yes. I ran your PR locally and that's how I noticed this.